_posts/2024-01-30-jump_year_in_review.md

```diff
@@ -0,0 +1,356 @@
+---
+layout: page
+title:  "JuMP: the year in review (2023)"
+date:   2024-01-30
+---
+
+_Author: Oscar Dowson_
```

Comment:
There's probably a metadata field for the author.

Reply:
There isn't. We could add a new layout, but maybe that's a separate PR.

Reply:
See https://github.com/jump-dev/jump-dev.github.io/pull/118